pootle/apps/pootle_misc/checks.py

```diff
@@ -48,6 +48,7 @@
     'functions': _(u"Functions"),
     'gconf': _(u"GConf values"),
     'isfuzzy': _(u"Fuzzy"),
+    'isreview': _(u"Review"),
```

Comment:
I originally added it to make the renderer work. I then asked if this was ok as it wasnt clear if there would be any side effects - and @dwaynebailey asked me to add it to the excluded checks.

I am happy to remove it - but just want to be sure that its not otherwise useful - ie that the title is not rendered anywhere.